### ISSUE TITLE
Cant compile due to duplicate string values

### ISSUE THREAD

**Issue by seanyc4** [valkriaine/Bouncy#15]
App wont compile due to duplicate string values:

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <!--<declare-styleable name="BouncyNestedScrollView">
        <attr format="float" name="overscroll_animation_size"/>
        <attr format="float" name="fling_animation_size"/>

        <attr format="integer" name="damping_ratio">
            <enum name="DAMPING_RATIO_NO_BOUNCY" value="0"/>
            <enum name="DAMPING_RATIO_LOW_BOUNCY" value="1"/>
            <enum name="DAMPING_RATIO_MEDIUM_BOUNCY" value="2"/>
            <enum name="DAMPING_RATIO_HIGH_BOUNCY" value="3"/>
        </attr>
        <attr format="integer" name="stiffness">
            <enum name="STIFFNESS_VERY_LOW" value="0"/>
            <enum name="STIFFNESS_LOW" value="1"/>
            <enum name="STIFFNESS_MEDIUM" value="2"/>
            <enum name="STIFFNESS_HIGH" value="3"/>
        </attr>
    </declare-styleable>-->
    <declare-styleable name="BouncyRecyclerView">
        <attr format="float" name="recyclerview_overscroll_animation_size"/>
        <attr format="float" name="recyclerview_fling_animation_size"/>
        <attr format="boolean" name="allow_drag_reorder"/>
        <attr format="boolean" name="allow_item_swipe"/>

        <attr format="integer" name="recyclerview_damping_ratio">
            <enum name="DAMPING_RATIO_NO_BOUNCY" value="0"/>
            <enum name="DAMPING_RATIO_LOW_BOUNCY" value="1"/>
            <enum name="DAMPING_RATIO_MEDIUM_BOUNCY" value="2"/>
            <enum name="DAMPING_RATIO_HIGH_BOUNCY" value="3"/>
        </attr>
        <attr format="integer" name="recyclerview_stiffness">
            <enum name="STIFFNESS_VERY_LOW" value="0"/>
            <enum name="STIFFNESS_LOW" value="1"/>
            <enum name="STIFFNESS_MEDIUM" value="2"/>
            <enum name="STIFFNESS_HIGH" value="3"/>
        </attr>
    </declare-styleable>
</resources>
```

I had to manually comment out the nestedscrolling values to get it to compile, please fix this.

**Comment by valkriaine** [comment]
This should be fixed now, please update to the newest version of Bouncy and try again, thank you!

**Comment by valkriaine** [comment]
resolved in 2.3
